### DISCUSSION TITLE
Move notification widget settings from "Widget & Live Wallpaper" section to "Notifications" section in the settings

### DISCUSSION THREAD

**Discussion by trymeouteh** [breezy-weather/breezy-weather#1176] [Ideas]
### Describe your idea

Move notification widget settings from "Widget & Live Wallpaper" section to "Notifications" section in the settings

### Other details

A minor suggestion. I think it is more logical to have these settings under notification instead of the widget settings.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.5 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
IMO the notification-widget is closer to a persistent widget in terms of rendering, customization, etc, compared to one-time notifications of alerts which you can find in the Notifications section.

Maybe we can find a better name than "notification-widget" that doesn't rely on the "notification" word instead?

**Reply by trymeouteh** [reply]
When I want to enable a persistent notification in an app. I expect it to be under the notification settings. Not a big deal if this change does not happen since it is really just a preference or an opinion on how to organize the settings.

**Reply by papjul** [reply]
> When I want to enable a persistent notification in an app

Do you have some examples?

**Reply by min7-i** [reply]
TodayWeather has such a feature where the settings are under notifications -> ongoing notification. _Admittedly, the app doesn't offer the possibility to configure widgets via the in-app settings. So it doesn't have a separate category for widgets there._

<details>

<summary>screenshots</summary>

The setting 'ongoing notification' says that it displays information in the status bar, but it also shows a notification in the notification shade.

<img src="https://github.com/breezy-weather/breezy-weather/assets/97251923/a1b3040b-c68c-494f-ab19-f382b87c0f80" width=50% />
<img src="https://github.com/breezy-weather/breezy-weather/assets/97251923/7c45616c-9b0f-43eb-8d81-8d111bcc4802" width=50% />

</details>

**Reply by papjul** [reply]
Yes, looks like a category "by default" in this case, since there is no widgets category. The result is that it feels a bit off in the middle of the other 4 options. A different section in the category would help the reading.
